### PR TITLE
Add clarification to OpenShift doc on custom route hostnames in OpenShift

### PR DESCRIPTION
**Pull request by BryanEllington**

### CODE REVIEW

**Comment by BryanEllington** [comment]
The only content change is the addition of this note to the OpenShift doc for both monitoring and logging:

"Note that OpenShift does not allow the use of the same across namespaces, so do not use the same hostname across logging and monitoring."

The other changes are just markdown formatting fixing linter warnings.

File: logging/OpenShift.md [hunk 1]
```diff
@@ -128,16 +155,24 @@ file (for only logging components) to use path-based routes.
 The hostnames for Kibana and Elasticsearch can be configured using
 `OPENSHIFT_ROUTE_HOST_KIBANA` and/or `OPENSHIFT_ROUTE_HOST_ELASTICSEARCH`
 (if the Elasticsearch route is enabled) in `$USER_DIR/user.env` or
-`$USER_DIR/logging/user.env`.
+`$USER_DIR/logging/user.env`. Note that OpenShift does not allow the use
+of the same across namespaces, so do not use the same hostname across
+logging and monitoring.
```
Comment (gsmith-sas) [inline logging/OpenShift.md:160]: same what?  Missing the word "hostname"?

File: monitoring/OpenShift.md [hunk 2]
```diff
@@ -133,7 +133,9 @@ Specify `OPENSHIFT_PATH_ROUTES=true` in the `$USER_DIR/user.env` file
 file (for only logging components) to use path-based routes.
 
 The Grafana hostname can be configured using `OPENSHIFT_ROUTE_HOST_GRAFANA` in
-`$USER_DIR/user.env` or `$USER_DIR/monitoring/user.env`.
+`$USER_DIR/user.env` or `$USER_DIR/monitoring/user.env`. Note that OpenShift
+does not allow the use of the same across namespaces, so do not use the same
```
Comment (gsmith-sas) [inline monitoring/OpenShift.md:137]: see comment above.

**Review by gsmith-sas — COMMENTED** [review]
I think you are missing a word...see comment.

File: monitoring/OpenShift.md (hunk 2 above)
Comment (BryanEllington) [reply monitoring/OpenShift.md]: Ugh...sorry. Not sure how that happened.

Fixed to: "Note that OpenShift does not allow the use of the same route hostname across namespaces, so do not use the same custom hostname across logging and monitoring."

**Review by gsmith-sas — APPROVED** [review]
lgtm

**Pull request merged**